common/lib/xmodule/xmodule/course_module.py

```diff
@@ -663,7 +663,14 @@ class CourseFields(object):
         # Ensure that courses imported from XML keep their image
         default="images_course_image.jpg"
     )
-
+    issue_badges = Boolean(
+        display_name=_("Issue OpenBadges"),
```

Comment:
@ottonomy Thank you for your comments. I also chatted with @antoviaque and it probably makes sense to retain the references to Open Badges, but corrected with the space between. 
So in lines 667, 669, please change "OpenBadges" to "Open Badges".